packages/manager/modules/sms/src/users/telecom-sms-users.html

```diff
@@ -99,8 +97,7 @@
                         data-property="alertThresholdInformations.support"
                         data-sortable>
                 <span data-ng-bind="::'sms_users_limit_user_support_' + $row.alertThresholdInformations.support | translate"
-                    data-uib-tooltip="{{ $row.alertThresholdInformations.alertEmail }}"
-                    data-tooltip-append-to-body="true">
```

Comment:
No impact ?

Reply:
Visually no but there was in `ng-if` required to show tooltip only when required. Have fixed this.